src/Abp.ZeroCore.EntityFrameworkCore/EntityHistory/EntityHistoryHelper.cs

```diff
@@ -216,8 +216,10 @@ private ICollection<EntityPropertyChange> GetPropertyChanges(EntityEntry entityE
                     continue;
                 }
 
-                var shouldSaveProperty = property.IsShadowProperty() ||
-                                         (IsAuditedPropertyInfo(property.PropertyInfo) ?? !auditedPropertiesOnly);
+                var shouldSaveProperty = property.IsShadowProperty()
```

Comment:
Shall we leave a comment for some context?

```suggestion
                var shouldSaveProperty = property.IsShadowProperty() // i.e. property.PropertyInfo == null
```